src/sagas/featureToggle.js

```diff
@@ -93,7 +92,7 @@ export function* monitorFeatureFlags(currentRetry = 0) {
   const { appVersion } = VersionNumber;
 
   const options = {
-    userId: getUniqueId(),
+    userId: 'HathorDemo',
```

Comment:
thought: I believe we have an opportunity here, we can create some constants of userId and configure our toggle features on unleash for them. It should be useful in development, but also in QA tests.